redbox-core/redbox/retriever/queries.py

```diff
@@ -117,18 +144,64 @@ def get_some(
         },
     }
 
-
-def get_metadata(
-    chunk_resolution: ChunkResolution | None,
-    state: RedboxState,
-) -> dict[str, Any]:
-    query_filter = build_query_filter(
-        selected_files=state["request"].s3_keys,
-        permitted_files=state["request"].permitted_s3_keys,
-        chunk_resolution=chunk_resolution,
-    )
-
-    return {
-        "_source": {"excludes": ["*embedding", "text"]},
-        "query": {"bool": {"must": {"match_all": {}}, "filter": query_filter}},
-    }
+    def scale_score(score: float, old_min: float, old_max: float, new_min=1.1, new_max: float = 2.0):
+        """Rescales an Elasticsearch score.
+
+        Intended to turn the score into a multiplier to weight a Gauss function.
+
+        If the old range is zero or undefined, returns new_min.
+        """
+        if old_max == old_min:
+            return new_min
+
```

Comment:
Should this be nested in such a big function?

Reply:
Makes the code a bit hard to follow imo as it is a huge stretch of one level indention

Reply:
I'll approve as deffo just a style comment. Substance looks great